docker-compose.yml

```diff
@@ -5,13 +5,13 @@ version: "3.1"
 # that is started up and polled
 services:
   kbase_blobstore:
-    image: kbase/blobstore:latest
+    build: .
     ports:
       - "8080:8080"
     environment:
       blobstore_host: 0.0.0.0:8080
-      mongo_host: localhost:27017
-      mongo_database: dc_blobstore_test
+      mongodb_host: mongo:27017
+      mongodb_database: dc_blobstore_test
```

Comment:
huh, these 2 are missed before? I assume they should map to whatever in deployment.cfg.templ.

Reply:
The compose file only worked because the default value for mongodb_host in the config template is `ci-mongo`. When I changed that to just `mongo` in the compose file the server couldn't connect and I discovered these were wrong